app/views/posts/show.html.erb

```diff
@@ -1,12 +1,11 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Blog App</title>
-</head>
-<body>
-<h1>Post's details</h1>
-</body>
-</html>
\ No newline at end of file
+<div class="card m-3">
+  <span class="ms-auto">Comments: <%= @post.comments_counter %> | Likes: <%= @post.likes_counter %></span>
+  <div class="card-body">
+    <h5 class="card-title"><%= @post.title %></h5>
+    <p class="card-text"><%= @post.text || '...' %></p>
+  </div>
+</div>
+
+<ul class="col p-0">
+  <%= render @post.comments %>
+</ul>
```

Comment:
- [ ] kindly make sure that the post page matches the wireframe 🙏 

your design 

![image](https://user-images.githubusercontent.com/89282221/222141745-12b5dd3c-bac9-41e4-8b4e-1a351d224025.png)

wireframe 
![image](https://user-images.githubusercontent.com/89282221/222141800-9f9a909c-e6e4-40a9-8407-b7fd33344ec2.png)

Reply:
- [ ] @frnega kindly ensure your design matches the wireframe given as previously requested, your comments design doesn't match the wireframe given.